templates/components/faceted-search/facets/hierarchy.html

```diff
@@ -46,7 +27,7 @@ <h5 class="accordion-title">
         </ul>
 
         {{#if show_more_toggle}}
-            <a href="#facetedSearch-navList--{{#replace '&' (hyphenate facet) }}{{else}}{{hyphenate facet}}{{/replace}}" role="button" class="toggleLink">
+            <a href="#facetedSearch-navList--{{dashcase facet}}" role="button" class="toggleLink">
```

Comment:
@lord2800 will this break things for facets which have `&` ?

Reply:
@junedkazi , Nope! IMO, dashcase is the correct helper to use here as it strips special characters and converts everything into a nice, usable form.

Reply:
Yeah, I wish I'd found that when we were initially looking into that problem. 😢